packages/client/src/object/convertWireToOsdkObjects/createOsdkObject.ts

```diff
@@ -140,50 +76,65 @@ export function createOsdkObject<
   objectDef: Q,
   rawObj: OntologyObjectV2,
 ): Osdk<ObjectTypeDefinition, any> {
-  // We use multiple layers of prototypes to maximize reuse and also to keep
-  // [RawObject] out of `ownKeys`. This keeps the code in the proxy below simpler.
-  const objectHolderPrototype = Object.create(
-    objectPrototypeCache.get(client, objectDef),
-    {
-      [RawObject]: {
-        value: rawObj,
-        writable: true, // so we can allow updates
-      },
-      [CachedPropertiesRef]: {
-        value: new Map<string | symbol, any>(),
-        writable: true,
-      },
+  // updates the object's "hidden class/map".
+  Object.defineProperties(rawObj, {
+    [ObjectDefRef]: { value: objectDef, enumerable: false },
+    [ClientRef]: { value: client, enumerable: false },
+    [RawObject]: {
```

Comment:
Nothing. But Im not trying to change ALL the code in the system to demonstrate JUST these objects being raw instead of proxies. The existing contract required both of these things, they just happen to be the same now.

If we go forward, this wouldn't be kept like this, we would clean it all up. But redoing the contract for interfaces would just muddy up this change (and I might as well just remove all proxies at that point since it would be far reaching). 

The goal here was to show that the objects could not be proxies and what that would be like (roughly) so we could discuss taking the next steps.